sympy/sets/tests/test_fancysets.py

```diff
@@ -59,7 +59,7 @@ def test_integers():
 def test_ImageSet():
     assert ImageSet(Lambda(x, 1), S.Integers) == FiniteSet(1)
     assert ImageSet(Lambda(x, y), S.Integers) == FiniteSet(y)
-
+    z = Interval(3,7)*Interval(8,11)*Interval(5,9)
```

Comment:
from what I see in this diff, it looks like this definition is not made in the right test function...it should perhaps be moved down to just before the line that uses it. And use a different name like `cube` or `c3` rather than the already imported `z`. That should help.